app/models/manager.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: wyeworkers
+#
+#  id         :integer          not null, primary key
+#  name       :string           not null
+#  type       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_wyeworkers_on_name  (name) UNIQUE
+#
+class Manager < Wyeworker
```

Comment:
I feel in that in the same way we have manager type, we should have a type per each role and let wyeworker to be an "abstract" class. That way would help to avoid confusions

Reply:
Alright, I understand! Will turn Wyeworker into BaseWyeworker, and then subtype it as Manager and Developer

Reply:
sounds good to me! I think that just Wyeworker is a good name for the base class actually